pkgs/servers/rainloop/default.nix

```diff
@@ -16,11 +16,23 @@
         sha256 = sha256;
       };
 
+      includeScript = ''
+        <?php
+
+        /**
+         * @return string
+         */
+        function __get_custom_data_full_path()
+        {
+          return '${dataPath}'; // custom data folder path
+        }
+      '';
+
       installPhase = ''
         mkdir $out
         cp -r rainloop/* $out
         rm -rf $out/data
-        ln -s ${dataPath} $out/data
+        echo -n "$includeScript" > $out/include.php
```

Comment:
Probably better to `pkgs.writeText` the include script and copy it here. This way, we don't have to think about shell escaping.

Reply:
@dasJ, I changed the derivation according to your advice. I used `cp` to copy the file. `ln -s` was also an option but not sure if it was best practice.